service_capacity_modeling/models/org/netflix/evcache.py

```diff
@@ -361,8 +361,6 @@ def capacity_plan(
     ) -> Optional[CapacityPlan]:
         # (Arun) EVCache defaults to RF=3 for tier 0 and tier 1
         default_copies = 3
```

Comment:
I think this should come from region context, no point in running say 3 copies if we only have 2 zones

Reply:
Yup, done